releasenotes/notes/synth_permutation_depth_lnn_kms-c444f3a363f3a903.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade_synthesis:
+  - |
```

Comment:
This should be labeled as a feature note, please see https://github.com/Qiskit/qiskit/pull/12715.